iocage.docset/Contents/Resources/Documents/install.html

```diff
@@ -58,7 +58,7 @@ <h2>Using github<a class="headerlink" href="#using-github" title="Permalink to t
 <p>If installing from github, the FreeBSD source tree <strong>must</strong> be located
 at <code class="samp docutils literal notranslate"><span class="pre">$SRC_BASE</span></code> ( <code class="samp docutils literal notranslate"><span class="pre">/usr/src</span></code> by default).</p>
 <p>To install from github, run these commands:</p>
-<p><code class="samp docutils literal notranslate"><span class="pre">pkg</span> <span class="pre">install</span> <span class="pre">python36</span> <span class="pre">git-lite</span> <span class="pre">libgit2</span> <span class="pre">cython3</span> <span class="pre">py36-pip</span></code></p>
+<p><code class="samp docutils literal notranslate"><span class="pre">pkg</span> <span class="pre">install</span> <span class="pre">python37</span> <span class="pre">git-lite</span> <span class="pre">libgit2</span> <span class="pre">cython3</span> <span class="pre">py36-pip</span></code></p>
```

Comment:
Same remark here for `libgit2` and `cython3` and there's an occurrence left to 36 in `py36-pip`